typescript/phoenix-sdk/psm-src/instructions/AddDesignatedMarketMaker.ts

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
`psm-src/index.ts` has a manual edit because it contains an import from a `types` directory which is not needed. 

Note: Types directory is generated because a struct was needed for shank but unused by the program.